nanoFramework.AtomMatrix/PixelController.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using nanoFramework.Hardware.Esp32.Rmt;
+using System.Drawing;
+
+namespace nanoFramework.AtomMatrix
+{
+    /// <summary>
+    /// Pixel controller for the WS2812C matrix.
+    /// </summary>
+    public class PixelController
```

Comment:
Stupid question: can this be used as well for the AtomLite? How this is different from the NeoPixel one?

Reply:
This one has a matrix of those. I can look into reusing the component there. Not sure if it's the same LED.